uptrain/operators/language/openai_evals.py

```diff
@@ -127,18 +129,17 @@ def run(self, data: t.Optional[pl.DataFrame] = None) -> TYPE_OP_OUTPUT:
         if path_samples_file is not None:
             os.remove(path_samples_file)
 
-        try:
-            output_data = pl.from_dicts(recorder.get_list_events("match"))
-        except:
-            output_data = pl.DataFrame()
-
+        unique_types = list(np.unique(np.array([x['type'] for x in recorder.get_list_events()])))
+        type_data = dict(zip(unique_types, [pl.from_dicts(x['data'] for x in recorder.get_list_events(type)) for type in unique_types]))
+        extra = {
+            "all_events": recorder.get_list_events(),
+            "run_data": recorder.get_run_data(),
+            "final_report": to_py_types(final_report),
+        }
+        extra.update(type_data)
         return {
-            "output": output_data,  # events are of multiple kinds. Undecided how to handle them in a consistent schema yet
-            "extra": {
-                "all_events": recorder.get_list_events(),
-                "run_data": recorder.get_run_data(),
-                "final_report": to_py_types(final_report),
-            },
+            "output": None,  # events are of multiple kinds. Undecided how to handle them in a consistent schema yet
```

Comment:
How do we store the output of this operator if it is None? SimpleCheck doesn't check for data in the "extra" key.

Reply:
So, the output of this operator is a bit complicated as it has two kinds of information - 
1) Prompt-Response
2) Matching/Metrics calculation
For now, updated output to reflect both the information. Ideally, this operator should not be used in SimpleCheck?

Reply:
Makes sense. It can be called an `OpenaiCheck` or something.